pandapower/io_utils.py

```diff
@@ -746,6 +748,23 @@ def from_json(cls, json_string):
         d = json.loads(json_string, cls=PPJSONDecoder)
         return cls.from_dict(d)
 
+    def __eq__(self, other):
+        """
+        isinstance: so that we know we can use to_json()
+        comparing class name instead of class directly allows more flexibility,
```

Comment:
Do you mean class attributes?

Reply:
With this, I meant that we don't compare the module here, we are comparing the class name and all the attributes. 
We do not compare methods.